tests/main.test.js

```diff
@@ -145,6 +145,15 @@ test('form elements', async () => {
   expect(finalCss).toMatch('option:selected');
 });
 
+test('invalid css', async () => {
+  expect.assertions(1);
+  try {
+    await runMinimalcss('invalid-css');
+  } catch (error) {
+    expect(error).toBeDefined();
```

Comment:
Maybe we can create assertion for the message of the error. WDYT?

Reply:
Yes, I'll check that instead.